src/cio_http_server.c

```diff
@@ -109,16 +110,16 @@ static const char *get_response_statusline(enum cio_http_status_code status_code
 {
 	switch (status_code) {
 	case CIO_HTTP_SWITCHING_PROTOCOLS:
-		return CIO_HTTP_VERSION " 101 Switching Protocols" CIO_CRLF "Server: cio http/" CIO_VERSION CIO_CRLF;
+		return CIO_HTTP_VERSION " 101 Switching Protocols" CIO_CRLF HTTP_SERVER_ID CIO_VERSION CIO_CRLF;
 	case CIO_HTTP_STATUS_OK:
-		return CIO_HTTP_VERSION " 200 OK" CIO_CRLF "Server: cio http/" CIO_VERSION CIO_CRLF;
+		return CIO_HTTP_VERSION " 200 OK" CIO_CRLF HTTP_SERVER_ID CIO_VERSION CIO_CRLF;
 	case CIO_HTTP_STATUS_BAD_REQUEST:
-		return CIO_HTTP_VERSION " 400 Bad Request" CIO_CRLF "Server: cio http/" CIO_VERSION CIO_CRLF;
+		return CIO_HTTP_VERSION " 400 Bad Request" CIO_CRLF HTTP_SERVER_ID CIO_VERSION CIO_CRLF;
 	case CIO_HTTP_STATUS_NOT_FOUND:
-		return CIO_HTTP_VERSION " 404 Not Found" CIO_CRLF "Server: cio http/" CIO_VERSION CIO_CRLF;
-
+		return CIO_HTTP_VERSION " 404 Not Found" CIO_CRLF HTTP_SERVER_ID CIO_VERSION CIO_CRLF;
+	case CIO_HTTP_STATUS_INTERNAL_SERVER_ERROR:
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant "case" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS3458)